### ISSUE TITLE
Constructor parameters in internal classes get prefixed.

### ISSUE THREAD

**Issue by zbynek** [timocov/ts-transformer-properties-rename#13]
**Bug report**

**Input code**

```ts
class Measure {
    height: number;

    constructor(public width: number, public ascent: number, public descent: number) {
        this.height = ascent + descent;
    }
}

export class Foo {}
```

**Expected output**

```ts
class Measure {
      constructor(width, ascent, descent) {
              this._internal_width = width;
              this._internal_ascent = ascent;
              this._internal_descent = descent;
              this._internal_height = ascent + descent;
       }
}

export class Foo {
}
```

**Actual output**

```ts
class Measure {
      constructor(_internal_width, _internal_ascent, _internal_descent) {
              this._internal_width = _internal_width;
              this._internal_ascent = _internal_ascent;
              this._internal_descent = _internal_descent;
              this._internal_height = ascent + descent;
       }
}

export class Foo {
}
```

**Additional context**
Reproducible if the code above is added as `input.ts` to the test directory in this repo and `npm run test` is started.

**Comment by timocov** [comment]
I confirm it's a bug.

**Comment by timocov** [comment]
The fix is just published in 0.7.0 version.
